bluepyopt/ephys/morphologies.py

```diff
@@ -199,16 +210,16 @@ def replace_axon(sim=None, icell=None):
         sim.neuron.h.execute('create axon[2]', icell)
 
         for index, section in enumerate(icell.axon):
-            section.nseg = 1
-            section.L = 30
+            section.L = axon_stub_length / 2
+            section.nseg = 1 + 2 * int(section.L / axon_nseg_frequency)
             section.diam = ais_diams[index]
             icell.axonal.append(sec=section)
             icell.all.append(sec=section)
 
         icell.axon[0].connect(icell.soma[0], 1.0, 0.0)
         icell.axon[1].connect(icell.axon[0], 1.0, 0.0)
 
-        logger.debug('Replace axon with AIS')
+        logger.debug(f"Replace axon with AIS, {axon_stub_length = }")
```

Comment:
The linting tests are failing again. This time, they expect no space around the = sign here -> {axon_stub_length=}
https://github.com/BlueBrain/BluePyOpt/actions/runs/10508468613/job/29113873731?pr=496#step:5:49

Reply:
Sure.